task/rpms-signature-scan/0.1/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Do we really need customization here?

Reply:
I took example from other tasks (such as buildah), but I now see that other tasks doesn't have it.
I will remove it

Reply:
Done